js/views/components/Moderators.js

```diff
@@ -331,6 +317,27 @@ export default class extends baseVw {
     return modCard;
   }
 
+  modShouldRender(model) {
+    const hideOnUnverified = this.getState().showVerifiedOnly && !model.isVerified;
+    const showCur = this.getState().showOnlyCur;
+    const hideOnCur = showCur && !model.hasModCurrency(showCur);
+    return !(hideOnUnverified || hideOnCur);
+  }
+
+  batchCardRender(model) {
+    if (!model || !(model instanceof Moderator)) {
```

Comment:
This solves an issue where huge amounts of renders, mostly in the settings/store view, were locking things up while mods were arriving over the socket. It will batch the renders just enough that very large sets arriving together are rendered as a group, but stragglers aren't delayed too much.